Add M3 release notes (#1539)
akka/alpakka: docs/src/main/paradox/release-notes/1.0-M3.md

# Alpakka 1.0-M3

## Third milestone release for Alpakka 1.0

> Even more modules have been restructured to meet the preferred code layout as illustrated by the [Reference connector](https://doc.akka.io/docs/alpakka/current/reference.html).

Reviewer: ```suggestion
All modules are now restructured to meet the preferred code layout as illustrated by the [Reference connector](https://doc.akka.io/docs/alpakka/current/reference.html).
```

[*all closed issues*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A1.0-M3+label%3Ap%3Akudu)

### @ref:[MongoDB](../mongodb.md)

**Breaking API changes**

* refactored the connector to more closely follow Alpakka connector guidelines @github[#1082](#1082)

[*all closed issues*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A1.0-M3+label%3Ap%3Amongodb)

### @ref:[Mqtt Streaming](../mqtt-streaming.md)

* various bugfixes @github[#1489](#1489) @github[#1480](#1480) @github[#1440](#1440) @github[#1436](#1436)

[*all closed issues*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A1.0-M3+label%3Ap%3Amqtt-streaming)

### @ref:[OrientDB](../orientdb.md)

**Breaking API changes**

* refactored the connector to more closely follow Alpakka connector guidelines @github[#1085](#1085)

[*all closed issues*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A1.0-M3+label%3Ap%3Aorientdb)

### @ref:[Slick](../slick.md)

* upgraded Slick to 3.3.0 @github[#1487](#1487)

[*all closed issues*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A1.0-M3+label%3Ap%3Aslick)

> ### @ref:[Sprint Web](../spring-web.md)

Reviewer: ```suggestion
### @ref:[Spring Web](../spring-web.md)
```
